Design/organizing system

sholokhov09 opened the pull request:
Submitting the deliverables. If possible, you can test the very rough prototype in the Figma link I attached. Thanks.

sholokhov09 committed:
Create OrganizingSystem_Deliverables.md with content

Added an overview and various sections including navigation, editing books, creating collections, user flow, and components.

sholokhov09 committed:
Add images and clarify Figma file sharing

Added additional images related to the feature and clarified sharing limitations.

sholokhov09 committed:
Add OrganizingSystem_Deliverables_Tablet.md file

chrox commented:
<img width="2558" height="1550" alt="image" src="https://github.com/user-attachments/assets/5d7194f2-2dd2-44a2-9dbd-3eb19648948d" />

I really like the side bar of Apple Books. It's minimal and elegant. Could we remove this, it looks a little distracting:

<img width="728" height="1024" alt="image" src="https://github.com/user-attachments/assets/ec0323da-ef62-4d0d-b433-feed5e6e8d55" />

sholokhov09 commented:
> <img width="2558" height="1550" alt="image" src="https://github.com/user-attachments/assets/5d7194f2-2dd2-44a2-9dbd-3eb19648948d" />
> 
> 
> 
> I really like the side bar of Apple Books. It's minimal and elegant. Could we remove this, it looks a little distracting:
> 
> 
> 
> <img width="728" height="1024" alt="image" src="https://github.com/user-attachments/assets/ec0323da-ef62-4d0d-b433-feed5e6e8d55" />
> 
> 
> 
> 
> 
> 

We can do that, but then we would need to figure out where to put the tools. Would you mind having it as items in List like Apple Books? That would be neater.

Another solution is to make the icons in a lighter grey to dim them down.

chrox commented:
> We can do that, but then we would need to figure out where to put the tools. Would you mind having it as items in List like Apple Books? That would be neater.

New collection button can be the last item of the collections list just as Apple Book does.

sholokhov09 commented:
Hi, I've addressed the feedback and also conducted some user testing to see if there's any blind spots. 

<img width="3024" height="1376" alt="Frame 78 (1)" src="https://github.com/user-attachments/assets/25a878a3-b98f-411a-93f7-efe2eccf09dd" />
<img width="3024" height="1376" alt="Frame 77 (1)" src="https://github.com/user-attachments/assets/c4378d70-b424-4023-9620-9c40e64870f0" />

sholokhov09 commented:
<img width="1744" height="750" alt="Frame 82 (1)" src="https://github.com/user-attachments/assets/b303631f-5d0a-42ab-adf1-c13d238f539b" />


Another Collection Color picking UI proposal.

chrox commented:
Solution 1 feels too much Material UI-ish, and I’m not a fan. The icon is unclear, and there isn’t enough context to convey what the action actually does. Material UI makes it easy to stack on more functions without really thinking through the flow or the use cases but that’s exactly why we need to be more deliberate here.

chrox commented:
> Another Collection Color picking UI proposal.

How exactly are the Collection colors meant to be used? Do we really need to display colored tags for each collection in every book’s overview? I feel like that would make the interface crowded and visually unappealing and it won’t feel minimal.

chrox commented:
Many apps, such as Calibre and KOReader, overwhelm users by dumping all their features upfront. But Readest isn’t going to be one of them. We will take a cleaner, more thoughtful approach.

sholokhov09 commented:
Hello, thank you so much for clarifying. I'm sending you the iterated design with Solution 2.

<img width="4312" height="1984" alt="Frame 81 (1)" src="https://github.com/user-attachments/assets/52e0f07e-2470-47be-b6bf-0daa64993dfd" />

About the Change color, it is mainly to organize how things look on the Sidebar. Collections of the same color will get clustered to each other on the Sidebar, but I don't think this feature has very high priority so feel free to leave it out.

<img width="590" height="856" alt="Frame 91 (1)" src="https://github.com/user-attachments/assets/1139d577-a63a-43fa-be1b-8493fc590735" />

chrox commented:
Thanks. I think we can go with the latest iteration.
